src/detect-tls.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2011-2012 ANSSI
+ * Copyright (C) 2011-2022 ANSSI
```

Comment:
strange update? Would the proper thing to do here not be that we add a second like for OISF?

Reply:
I'll revert this change and add an OISF copyright for 2022